docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3335,21 +3326,76 @@ function toggleBreakout() {
   }
 }
 
-async function toggleBrb() {
+async function stopScreenShare() {
+  console.log('MeetingControls#stopScreenShare()');
+  try {
+    if (localMedia.screenShare.audio) {
+      localMedia.screenShare.audio?.stop();
+    }
+    if (localMedia.screenShare.video) {
+      localMedia.screenShare.video?.stop();
+    }
+
+    console.log('MeetingControls#stopScreenShare() :: Successfully stopped sharing!');
+  }
+  catch (error) {
+    console.log('MeetingControls#stopScreenShare() :: Error stopping screen share!');
+    console.error(error);
+  }
+}
+
+async function toggleBrb({unmuteOnlyAudio = false, unmuteOnlyVideo = false}) {
```

Comment:
Technically someone could set both `unmuteOnlyAudio` and `unmuteOnlyVideo` to true, which makes the word "only" in the variable names a bit strange. I'd rename them to simply `unmuteAudio` and `unmuteVideo`.